packages/components/src/AnimatedSwitcher/AnimatedSwitcher.stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Button } from "@jobber/components/Button";
+import { Text } from "@jobber/components/Text";
+import { AnimatedSwitcher } from ".";
+
+<Meta title="Components/AnimatedSwitcher" component={AnimatedSwitcher} />
+
+# AnimatedSwitcher
+
+Smoothly switch between two components that aim to add delight when the user
```

Comment:
Love this description

Now that we have a better idea of documentation sections moving forward should we include some of the relevant ones?  I think Content Guidelines and/or Design and usage guidelines to be a bit more clear about how this is best used between buttons/icons

Reply:
not at this moment. It's as simple as it goes so I don't think we need one just yet